hazelcast-client-new/src/main/java/com/hazelcast/client/proxy/ClientQueueProxy.java

```diff
@@ -64,21 +64,22 @@
 import java.util.Set;
 import java.util.concurrent.TimeUnit;
 
-/**
- * @author ali 5/19/13
- */
 public final class ClientQueueProxy<E> extends ClientProxy implements IQueue<E> {
```

Comment:
Probably the onInitialize and invoke logic could be pulled into a super class. E.g. a PartitionSpecificClientProxy.